community/modules/file-system/DDN-EXAScaler/outputs.tf

```diff
@@ -24,16 +24,47 @@ output "ssh_console" {
   value       = module.ddn_exascaler.ssh_console
 }
 
+output "client_config" {
```

Comment:
nit: The name makes me think it's a config file, not a script. I'd append _script to the name.

Disregard if you disagree.

Reply:
I agree, I was matching the ddn name but I think appending script satisfies both inclinations. Will update.

Reply:
Done.